apps/ui/src/views/App.vue

```diff
@@ -1,5 +1,5 @@
 <script setup lang="ts">
-import { simplifyURL } from '@/helpers/utils';
+import { getUrl, simplifyURL } from '@/helpers/utils';
```

Comment:
These changes does not seem to be related to this PR

Reply:
Right, this is to fix ecosystem app page I should have done another PR